examples/themes/winamp-theme.html

```diff
@@ -0,0 +1,426 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width" />
+    <title>Media Chrome Winamp Theme Example</title>
+    <script type="module" src="../../dist/index.js"></script>
```

Comment:
should this page duplicate media-theme-winamp.js? Or include it and then use the media-theme-winamp element?

Reply:
ah yes good call, I'll change that so it uses the web comp